api-reference/10 UI Components/GridBase/1 Configuration/pager/allowedPageSizes.md

```diff
@@ -9,4 +9,4 @@ default: 'auto'
 Specifies the available page sizes in the page size selector.
 
 ---
-The UI component determines the allowed page sizes depending on the amount of data by default.
\ No newline at end of file
+Set this property to an array of numbers that specify available page sizes. If you want to let users display all records on a single page, add the *"all"* value to this array. The default value *"auto"* means that the component automatically determines available page sizes based on the total number of records.
```

Comment:
- `If you want to let users display all records` -> If you want to allow users to display all records
- When the component is set to *"auto"* (the default value), the available page sizes is automatically calculated based on the total number of records.